templates/firefox-ios.xliff

```diff
@@ -2500,6 +2536,54 @@
+        <source>Firefox wallpaper, sunrise pattern, button</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the sunrise firefox wallpaper.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.Accessibility.ToggleButton" xml:space="preserve">
+        <source>Homepage wallpaper cycle toggle</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the accessibility string of the toggle for turning wallpaper cycling shortcut on or off on the homepage.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.CollectionTitle" xml:space="preserve">
+        <source>OPENING SCREEN</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the title of the section that allows users to change the wallpaper settings for the application.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.SwitchDescription" xml:space="preserve">
+        <source>Cycle through and update image without leaving the homepage.</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the string describing the switch button's function, which allows a user to toggle wallpaper switching from the homepage logo on or off.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.SwitchTitle" xml:space="preserve">
+        <source>Tap logo to change wallpaper</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the string titling the switch button's function, which allows a user to toggle wallpaper switching from the homepage logo on or off.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.Title" xml:space="preserve">
+        <source>Wallpaper</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the title of that screen, which allows users to change the wallpaper settings for the application.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.UpdatedToast" xml:space="preserve">
+        <source>Wallpaper Updated!</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the title of toast that comes up when the user changes wallpaper, which lets them know that the wallpaper has been updated.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.UpdatedToastButton" xml:space="preserve">
+        <source>View</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the title of the button found on the toast that comes up once the user changes wallpaper, and allows users to dismiss the settings page.</note>
```

Comment:
Could you just add that "View" is a verb here?
I understand that to be the case from the comment, but want to make sure all localizers get it :)
(because it can also be a noun when you see it on its own)

Reply:
Correct. It is referring to the action of viewing the thing. :)

Reply:
Ah thanks! Please add this detail into the comments :)